contrib/babelfishpg_tsql/src/multidb.c

```diff
@@ -1035,7 +1035,7 @@ rewrite_role_name(RoleSpec *role)
 {
 	char	   *cur_db = get_cur_db_name();
 
-	role->rolename = get_physical_user_name(cur_db, role->rolename, false);
+	role->rolename = get_physical_user_name(cur_db, role->rolename, false, false);
```

Comment:
if role->rolename is palloc copy then we should free it before reassignment

Reply:
need to pfree `cur_db` as well

Reply:
Done

Reply:
lets do this, 
```
char *temp_rolename = get_physical_user_name(cur_db, role->rolename, false, false);
pfree(role->rolename)
role->rolename = temp_rolename
```

Reply:
Done